### ISSUE TITLE
[Content Author Module 6: Multimedia] What are "governmental regulations and other policy requirements"?

### ISSUE THREAD

**Issue by MicheleAWilliams-A11y** [w3c/wai-curricula#549]
Under Planning Audio and Video, it says to "Explain that different types of audio and video content have different accessibility requirements based on governmental regulations and other policy requirements."

As someone who would be a user of this material, I'm not sure what is meant by "governmental regulations and other policy requirements". Would this be like the CVAA in the U.S.? Not sure why this is called out apart from WCAG; most regulations I can think of in the U.S. have more to do with television than the Web, for instance.

**Comment by eoncins** [comment]
In the case of EU there are three main European Directives addressing accessibility in ICT products, services and tools:
- European Accessibility Act (https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A32019L0882)
- Web Accessibility Directive (https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32016L2102)
- Audiovisual Media Service Directive (https://eur-lex.europa.eu/legal-content/EN/ALL/?uri=celex%3A32010L0013) also for Video on Demand (VoD) platforms
These three legislation pieces have to be transposed differently to the European Member states with the corresponding governmental regulations and other policy requirements.
In addition, web accessibility guidelines are different worldwide: Web Accessibility Laws & Policies (https://www.w3.org/WAI/policies/) maybe to cross-reference to this resource might be worth.

**Comment by daniel-montalvo** [comment]
> Under Planning Audio and Video, it says to "Explain that different types of audio and video content have different accessibility requirements based on governmental regulations and other policy requirements."
> 
> As someone who would be a user of this material, I'm not sure what is meant by "governmental regulations and other policy requirements". Would this be like the CVAA in the U.S.? Not sure why this is called out apart from WCAG; most regulations I can think of in the U.S. have more to do with television than the Web, for instance.

"governmental regulations and other policy requirements" is now changed to "government regulations and other policy requirements"

Does this help clarify in terms of language?

I think it is worth mentioning government regulations and other policy requirements, as there seems to be fragmentation for which requirements each of them has.

But it is a good question whether wwe should be agnostic in terms of other regulations and just focus on what we have in the [Making Audio and Video Media Accessible](https://www.w3.org/WAI/media/av/) resource.

**Comment by daniel-montalvo** [comment]
I actually found references to government regulations and policies in the [Planning Audio and VideoMedia](https://www.w3.org/WAI/media/av/planning/) subpage.

My proposal is to leave as-is. See also 

https://github.com/w3c/wai-curricula/issues/549#issuecomment-1167930023

Awarness thatin different regions theere might be different requirements depending on the policies in place is something I think we should have as a LO.

**Comment by MicheleAWilliams-A11y** [comment]
Sounds good to me - and I do like the change to "government regulations". It's minor but did make a difference.

**Comment by eoncins** [comment]
The new wording sound good to me too.

The main issue is to inform students/teachers that "government regulations" for accessibility are present in legislation, and each country has its own regulations.

Thanks,

Estel·la Oncins Noguer
Post Doctoral Research Fellow TransMedia Catalonia
Edifici MRA 126  - Campus UAB
08193 Bellaterra
Barcelona
T. +34 610 655 149

orcid.org/0000-0002-0291-3036<https://orcid.org/0000-0002-0291-3036><https://orcid.org/0000-0002-0291-3036>
http://grupsderecerca.uab.cat/transmedia<http://grupsderecerca.uab.cat/transmedia/>
<http://grupsderecerca.uab.cat/transmedia/>
<http://grupsderecerca.uab.cat/transmedia/>

<http://grupsderecerca.uab.cat/transmedia/><http://grupsderecerca.uab.cat/transmedia/>

________________________________

**Comment by daniel-montalvo** [comment]
Agreed in[TF meeting 12 July](https://www.w3.org/2022/07/12-wai-curricula-minutes.html)

Updated draft is at
https://content-author-modules--wai-curricula.netlify.app/curricula/content-author-modules/multimedia/#topic-planning-audio-and-video
